src/main/webapp/js/app/nearyouService.js

```diff
@@ -0,0 +1,23 @@
+function nearyouScript() {
+}
+nearyouScript.prototype.locationRadius = 2000;
+nearyouScript.prototype.events = [];
+nearyouScript.prototype.eventObjs = [];
+nearyouScript.prototype.findNearbyEvents = function(events, currLocation) {
+  'use strict';
+  if (events.length == 0) {
+    return "No nearby events";
+  }
+};
+
+nearyouScript.prototype.isNearby = function(eventObj, currLocation) {
+  
+  var distanceFromCurrLocation = Math.sqrt(
+    (Math.pow((eventObj.xlocation - currLocation.xlocation), 2)) + 
+    (Math.pow((eventObj.ylocation - currLocation.ylocation), 2)) 
+  );
```

Comment:
What are the units of `xlocation` and `ylocation`? Latitudes and longitudes?

Does this computation behave as expected with two nearby points that are across the International Date Line from one another?

Reply:
I took these to be simple x and y coordinates on a 2d plane. Do you think it will be better to make them latitudes and longitudes? Then I will have to figure out how to make them behave as expected with points across the International Date Line